### DISCUSSION TITLE
Unable to Loop Out

### DISCUSSION THREAD

**Discussion by emretezel** [Ride-The-Lightning/RTL#782] [Troubleshooting Questions] [answered]
Hi Everyone,

Running RTL inside Umbrel on Ubuntu, tried to loop out yesterday, says process initiated but nothing happened in 24 hours with no error message. Please see details below. Is safe to assume this has failed, or is it still trying. Will it ever update the status on the RTL loop tab?

Thank you

```
Status
 Initiated
Amount (Sats)
8,500,000
Initiation Time
31/Aug/2021 22:20
Last Update Time
31/Aug/2021 22:20
Server Cost (Sats)
0
Offchain Cost (Sats)
0
Onchain Cost (Sats)
0
ID
5OZE9RWT5whjBzATufbpwq3Ctjr7+pkWN/6SNBEu7ig=
HTLC Address
bc1qsrmygcw7v8djaev3g87jxtqzrk82xek0dr3t7vmapsed67dduhqs0s9aaw
```

**Comment by saubyk** [comment, marked as answer]
Hi @emretezel, couple of ways to verify if Loop out is still open:
1. Check the 'Active HTLCs' tab under 'Peers/Channels' menu. If there are HTLCs outstanding it implies that loop is still open and vice-versa.
2. Verify if you have any Payments made under the 'Payments' menu for the amount that you looped out for. If not, it implies that you're Loop transaction has not gone through.
3. Verify the status of the loop transaction on the 'Lightning Terminal' app on your Umbrel.
